fs/xfstests.py.data/nvdimm.yaml

```diff
@@ -1,20 +1,22 @@
 skip_dangerous: True
 scratch_mnt: '/mnt/scratch_pmem'
 test_mnt: '/mnt/test_pmem'
-mount_opt: '-o dax'
+test_range: "null"
```

Comment:
if we are looking to run entire xfs test suite so test_range param not needed

Reply:
in some cases we need test range param to run a particular or a range of tests, so i feel its fine

Reply:
but here we are passing this as null ?

Reply:
@disgoel are you thinking we can go with this ?

Reply:
@PraveenPenguin yes in my opinion we can go with this, if you are looking from CR point of view then I can remove this option

Reply:
yes it on CR point of you

Reply:
removed test_range parameter